### PR TITLE
[OPF] Add the MTK implementation as well

### PR DESCRIPTION
**Pull request by Vaibhavdixit02**

### CODE REVIEW

File: benchmarks/OptimizationFrameworks/optimal_powerflow.jmd [hunk 1]
```diff
@@ -2144,6 +2335,17 @@ function multidata_multisolver_benchmark(dataset_files; sizelimit = SIZE_LIMIT)
...
 
+        @info "Running ModelingToolkit.jl"
+        model, res = solve_opf_mtk(dataset)
+        push!(cases, split(file,"/")[end])
+        push!(vars, res["variables"])
+        push!(cons, res["constraints"])
+        push!(mtk_time, res["time_solve"])
+        push!(mtk_time_modelbuild, res["time_build"])
+        push!(mtk_time_compilation, res["time_solve_compilation"])
+        push!(mtk_cost, res["cost"])
```
Comment (ChrisRackauckas) [inline benchmarks/OptimizationFrameworks/optimal_powerflow.jmd:2346]: These variables aren't defined.

File: benchmarks/OptimizationFrameworks/optimal_powerflow.jmd [hunk 2]
```diff
@@ -2289,24 +2289,28 @@ function multidata_multisolver_benchmark(dataset_files; sizelimit = SIZE_LIMIT)
     cons = Int[]
 
     optimization_time = Float64[]
+    mtk_time = []
```
Comment (ChrisRackauckas) [inline benchmarks/OptimizationFrameworks/optimal_powerflow.jmd:2292]:
```suggestion
    mtk_time = Float64[]
```

File: benchmarks/OptimizationFrameworks/optimal_powerflow.jmd [hunk 3]
```diff
@@ -2144,6 +2339,17 @@ function multidata_multisolver_benchmark(dataset_files; sizelimit = SIZE_LIMIT)
         push!(optimization_time_compilation, res["time_solve_compilation"])
         push!(optimization_cost, res["cost"])
 
+        @info "Running ModelingToolkit.jl"
+        model, res = solve_opf_mtk(dataset)
+        push!(cases, split(file,"/")[end])
+        push!(vars, res["variables"])
+        push!(cons, res["constraints"])
+        push!(mtk_time, res["time_solve"])
```
Comment (ChrisRackauckas) [inline benchmarks/OptimizationFrameworks/optimal_powerflow.jmd:2347]:
```suggestion
        model, res = solve_opf_mtk(dataset)
        push!(mtk_time, res["time_solve"])
```

**Review by ChrisRackauckas — APPROVED** [review]

File: benchmarks/OptimizationFrameworks/pglib_opf_case73_ieee_rts.m [hunk 4]
```diff
@@ -0,0 +1,755 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```
Comment (ChrisRackauckas) [inline benchmarks/OptimizationFrameworks/pglib_opf_case73_ieee_rts.m:1]: why is this one added?

Comment (Vaibhavdixit02) [reply benchmarks/OptimizationFrameworks/pglib_opf_case73_ieee_rts.m:1]: Ah shit, was for local testing added it by mistake

**Pull request merged**